### ISSUE TITLE
Adding hosts stuck in "provisioning" state.

### ISSUE THREAD

**Issue by sangeethah** [rancher/rancher#8329]
Rancher server version - Build from wagyu

Steps to reproduce the problem:
Added 3 Digitial Ocean hosts.
Out of the 3 hosts added , 1 of the host is stuck in ""provisioning" state forever.
Following error seen repeatedly  in rancher-server logs: 
```
time="2017-03-28T18:20:43Z" level=info msg="Activating Machine" eventId=f58c9d8e-dc67-415a-aad9-7cf5dbd42d5f resourceId=1ph1 
time="2017-03-28T18:20:43Z" level=error msg="Could not rewrap secret" 
time="2017-03-28T18:20:43Z" level=error msg="Error in request, code : 400: Signatures did not match" 
2017-03-28 18:20:43,053 ERROR [:] [] [] [] [tp192881625-118] [i.c.p.d.m.a.f.MachineOutputFilter   ] Failed to decrypt machine extracted config java.io.IOException: Failed to rewrap secret :Bad Request
	at io.cattle.platform.framework.secret.SecretsServiceImpl$3.handleResponse(SecretsServiceImpl.java:162) ~[cattle-framework-encryption-0.5.0-SNAPSHOT.jar:na]
	at io.cattle.platform.framework.secret.SecretsServiceImpl$3.handleResponse(SecretsServiceImpl.java:157) ~[cattle-framework-encryption-0.5.0-SNAPSHOT.jar:na]
	at org.apache.http.client.fluent.Response.handleResponse(Response.java:85) ~[fluent-hc-4.3.1.jar:4.3.1]
	at io.cattle.platform.framework.secret.SecretsServiceImpl.decrypt(SecretsServiceImpl.java:157) ~[cattle-framework-encryption-0.5.0-SNAPSHOT.jar:na]
	at io.cattle.platform.docker.machine.api.filter.MachineOutputFilter.filter(MachineOutputFilter.java:32) ~[cattle-docker-machine-0.5.0-SNAPSHOT.jar:na]
	at io.github.ibuildthecloud.gdapi.response.impl.ResourceOutputFilterChain.filter(ResourceOutputFilterChain.java:23) [cattle-framework-java-server-0.5.0-SNAPSHOT.jar:na]
	at io.github.ibuildthecloud.gdapi.request.resource.impl.AbstractBaseResourceManager.convertResponse(AbstractBaseResourceManager.java:200) [cattle-framework-java-server-0.5.0-SNAPSHOT.jar:na]
	at io.github.ibuildthecloud.gdapi.request.resource.impl.FilteredResourceManager.convertResponse(FilteredResourceManager.java:76) [cattle-framework-java-server-0.5.0-SNAPSHOT.jar:na]
	at io.github.ibuildthecloud.gdapi.response.ResponseObjectConverter.handle(ResponseObjectConverter.java:37) [cattle-framework-java-server-0.5.0-SNAPSHOT.jar:na]
	at io.github.ibuildthecloud.gdapi.servlet.ApiRequestFilterDelegate.doFilter(ApiRequestFilterDelegate.java:99) [cattle-framework-java-server-0.5.0-SNAPSHOT.jar:na]
	at io.cattle.platform.api.servlet.ApiRequestFilter$1.runInContext(ApiRequestFilter.java:96) [cattle-framework-api-0.5.0-SNAPSHOT.jar:na]
	at org.apache.cloudstack.managed.context.ManagedContextRunnable$1.run(ManagedContextRunnable.java:49) [cattle-framework-managed-context-0.5.0-SNAPSHOT.jar:na]
	at org.apache.cloudstack.managed.context.impl.DefaultManagedContext$1.call(DefaultManagedContext.java:55) [cattle-framework-managed-context-0.5.0-SNAPSHOT.jar:na]
	at org.apache.cloudstack.managed.context.impl.DefaultManagedContext.callWithContext(DefaultManagedContext.java:108) [cattle-framework-managed-context-0.5.0-SNAPSHOT.jar:na]
	at org.apache.cloudstack.managed.context.impl.DefaultManagedContext.runWithContext(DefaultManagedContext.java:52) [cattle-framework-managed-context-0.5.0-SNAPSHOT.jar:na]
	at org.apache.cloudstack.managed.context.ManagedContextRunnable.run(ManagedContextRunnable.java:46) [cattle-framework-managed-context-0.5.0-SNAPSHOT.jar:na]
	at io.cattle.platform.api.servlet.ApiRequestFilter.doFilter(ApiRequestFilter.java:106) [cattle-framework-api-0.5.0-SNAPSHOT.jar:na]
	at org.eclipse.jetty.servlet.ServletHandler$CachedChain.doFilter(ServletHandler.java:1652) [jetty-servlet-9.2.11.v20150529.jar:9.2.11.v20150529]
	at org.eclipse.jetty.servlets.UserAgentFilter.doFilter(UserAgentFilter.java:83) [jetty-servlets-9.2.11.v20150529.jar:9.2.11.v20150529]
	at org.eclipse.jetty.servlets.GzipFilter.doFilter(GzipFilter.java:364) [jetty-servlets-9.2.11.v20150529.jar:9.2.11.v20150529]
	at org.eclipse.jetty.servlet.ServletHandler$CachedChain.doFilter(ServletHandler.java:1652) [jetty-servlet-9.2.11.v20150529.jar:9.2.11.v20150529]
	at org.eclipse.jetty.servlet.ServletHandler.doHandle(ServletHandler.java:585) [jetty-servlet-9.2.11.v20150529.jar:9.2.11.v20150529]
	at org.eclipse.jetty.server.handler.ScopedHandler.handle(ScopedHandler.java:143) [jetty-server-9.2.11.v20150529.jar:9.2.11.v20150529]
	at org.eclipse.jetty.security.SecurityHandler.handle(SecurityHandler.java:577) [jetty-security-9.2.11.v20150529.jar:9.2.11.v20150529]
	at org.eclipse.jetty.server.session.SessionHandler.doHandle(SessionHandler.java:223) [jetty-server-9.2.11.v20150529.jar:9.2.11.v20150529]
	at org.eclipse.jetty.server.handler.ContextHandler.doHandle(ContextHandler.java:1127) [jetty-server-9.2.11.v20150529.jar:9.2.11.v20150529]
	at org.eclipse.jetty.servlet.ServletHandler.doScope(ServletHandler.java:515) [jetty-servlet-9.2.11.v20150529.jar:9.2.11.v20150529]
	at org.eclipse.jetty.server.session.SessionHandler.doScope(SessionHandler.java:185) [jetty-server-9.2.11.v20150529.jar:9.2.11.v20150529]
	at org.eclipse.jetty.server.handler.ContextHandler.doScope(ContextHandler.java:1061) [jetty-server-9.2.11.v20150529.jar:9.2.11.v20150529]
	at org.eclipse.jetty.server.handler.ScopedHandler.handle(ScopedHandler.java:141) [jetty-server-9.2.11.v20150529.jar:9.2.11.v20150529]
	at org.eclipse.jetty.server.handler.HandlerWrapper.handle(HandlerWrapper.java:97) [jetty-server-9.2.11.v20150529.jar:9.2.11.v20150529]
	at org.eclipse.jetty.server.Server.handle(Server.java:499) [jetty-server-9.2.11.v20150529.jar:9.2.11.v20150529]
	at org.eclipse.jetty.server.HttpChannel.handle(HttpChannel.java:310) [jetty-server-9.2.11.v20150529.jar:9.2.11.v20150529]
	at org.eclipse.jetty.server.HttpConnection.onFillable(HttpConnection.java:257) [jetty-server-9.2.11.v20150529.jar:9.2.11.v20150529]
	at org.eclipse.jetty.io.AbstractConnection$2.run(AbstractConnection.java:540) [jetty-io-9.2.11.v20150529.jar:9.2.11.v20150529]
	at org.eclipse.jetty.util.thread.QueuedThreadPool.runJob(QueuedThreadPool.java:635) [jetty-util-9.2.11.v20150529.jar:9.2.11.v20150529]
	at org.eclipse.jetty.util.thread.QueuedThreadPool$3.run(QueuedThreadPool.java:555) [jetty-util-9.2.11.v20150529.jar:9.2.11.v20150529]
	at java.lang.Thread.run(Thread.java:745) [na:1.8.0_72]
```

**Comment by sangeethah** [comment]
After this error , adding any host on the setup is stuck in "provisoning" state with the same error.

**Comment by cloudnautique** [comment]
This should be addressed in 1.5.4-rcx

**Comment by sangeethah** [comment]
Tested with build from master (Apr 17).
Not able to reproduce this issue when adding multiple Digital Ocean hosts at the same time.
